### DISCUSSION TITLE
Anyone doing "commissions"?

### DISCUSSION THREAD

**Discussion by mikedry** [elrumo/macOS_Big_Sur_icons_replacements#1107] [Icon request]
I have a handful (four) icons that I'd like to have "converted" into Big Sur-type icons, but have absolutely no skills in this respect.  Would be willing to pay a reasonable amount of $ to make it happen.

**Comment by w0lfschild** [comment]
Maybe check out 5er

https://www.fiverr.com/categories/graphics-design?source=category_tree

**Comment by elrumo** [comment]
I can look into it, you can email me the details on [elias.ruiz.monserrat@gmail.com ](mailto:elias.ruiz.monserrat@gmail.com )
